packages/manager/apps/dedicated/client/app/license/order/license-order.html

```diff
@@ -154,7 +154,7 @@ <h2 data-translate="license_order_step2_title"></h2>
                         class="form-control"
                         id="optionVersion"
                         name="optionVersion"
-                        data-ng-options="optionVersion as (('license_designation_' + selected.licenseType + '_version_' + optionVersion.value) | translate) for optionVersion in types[selected.licenseType].options[0].options"
+                        data-ng-options="optionVersion as optionVersion.displayName for optionVersion in formatedOptions"
```

Comment:
Can the translations keys used previously be remove then ? Or are they still used elsewhere ?

Reply:
these are used by other options. I have moved them from template to controller to handle them conditionally.